src/locales/ko/pokemon-summary.ts

```diff
@@ -0,0 +1,17 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales.js";
+
+export const pokemonSummary: SimpleTranslationEntries = {
+  "pokemonInfo": "포켓몬 정보",
+  "status": "능력치",
```

Comment:
```suggestion
  "status": "스테이터스",
```
The current version(9th gen SV) officially translates "status" into "스테이터스"